### PR TITLE
Fix for Female Human Silk Coats

### LINKED ISSUE #309: [Bug]: Silk Coat Typo disallowing human females to wear it.
### Checks

- [X] I have searched the bug with a few keywords, and I confirm this bug was not yet reported.

### Map

RogueTown, DakkaTown

### Describe the issue

ok so basically epic typo in the code that disallows female humans to wear the silk coat dress. 
image

### How to reproduce

try to put silk coat on as human female>you cant

### Round ID

N/A

### Staff ckey

N/A

### PR DESCRIPTION
**Pull request by AudiGzz**
Fixes female humans not being able to wear silk coats

Closes #309 

## Pre-Merge Checklist

- [X] You tested this on a local server.
- [X] This code did not runtime during testing.
- [X] You documented all of your changes.

### CODE REVIEW

File: code/modules/clothing/rogueclothes/armor.dm [hunk 1]
```diff
@@ -444,7 +444,7 @@
 	sellprice = 50
 	armor_class = ARMOR_CLASS_LIGHT
 	allowed_sex = list(FEMALE)
-	allowed_race = list("humen", "tiefling", "dwarfm","argonian", "elfd", "elfw", "helf", "aasimar")
+	allowed_race = list("human", "tiefling", "dwarfm","argonian", "elfd", "elfw", "helf", "aasimar")
```
Comment (ThePainkiller) [inline code/modules/clothing/rogueclothes/armor.dm:447]:
```suggestion
	allowed_race = ALL_RACES_LIST_NAMES
```
We have a proper list for all races, this eliminates meme races not implemented like we have

**Review by ThePainkiller — CHANGES_REQUESTED** [review]

File: code/modules/clothing/rogueclothes/armor.dm (hunk 1 above)
Comment (AudiGzz) [reply code/modules/clothing/rogueclothes/armor.dm]: Will keep in mind for the future.

**Review by ThePainkiller — APPROVED** [review]

File: code/modules/clothing/rogueclothes/armor.dm (hunk 1 above)
Comment (ThePainkiller) [reply code/modules/clothing/rogueclothes/armor.dm]: I have made a mistake....

File: code/modules/clothing/rogueclothes/armor.dm [hunk 2]
```diff
@@ -444,7 +444,7 @@
 	sellprice = 50
 	armor_class = ARMOR_CLASS_LIGHT
 	allowed_sex = list(FEMALE)
-	allowed_race = list("humen", "tiefling", "dwarfm","argonian", "elfd", "elfw", "helf", "aasimar")
+	allowed_race = ALL_RACES_LIST_NAMES
```
Comment (ThePainkiller) [inline code/modules/clothing/rogueclothes/armor.dm:447]:
```suggestion
	allowed_race = ALL_RACES_LIST
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]

**Review by ThePainkiller — APPROVED** [review]

**Pull request merged**